scout/server/blueprints/cases/views.py

```diff
@@ -280,7 +280,7 @@ def mt_report(institute_id, case_name):
             mimetype="application/zip",
             as_attachment=True,
             download_name="_".join(["scout", case_name, "MT_report", today]) + ".zip",
-            cache_timeout=0,
+            max_age=0,
```

Comment:
cache_timeout is renamed to max_age ---> https://github.com/pallets/flask/pull/4337

Reply:
"These old names were deprecated in Flask's wrappers. The deprecation period has been extended to 2.2." Those deprecation messages.. 😆